src/ShellApplication.cpp

```diff
@@ -58,6 +58,12 @@ ShellApplication::ShellApplication(int & argc, char ** argv, bool isMirServer)
 
     m_qmlArgs.setMode(parser.mode());
 
+    {
+        char buffer[200];
+        property_get("ubuntu.unity8.interactive_blur", buffer, "false");
```

Comment:
Should we define that rather in the ro.xyz space? As its not changeable at runtime

Reply:
It's not a security-relevant property, and for debugging purposes by the porter I decided to keep it in the same namespace as aethercast and platform-api booster properties are.